appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -147,6 +138,9 @@ class LoggedInFlowNode @AssistedInject constructor(
 
     override fun onBuilt() {
         super.onBuilt()
+
+        syncOrchestrator.start()
```

Comment:
I think this code will not be run if the app is killed and a Push is received. In this case the sync will not start?

Reply:
I think you're right, and I completely overlooked this... This complicates things, as we would have to use `SyncOrchestrator` in the `AppScope` and that means holding as many as `MatrixClient`s there are in `MatrixClientsHolder`, maybe following a similar approach 🫤 .